### DISCUSSION TITLE
Tracking World Quest Progress

### DISCUSSION THREAD

**Discussion by Gnichned** [SavedInstances/SavedInstances#826] [Q&A] [answered]
I'm looking for an addon, that tracks the gold-world quests of all my cahrs. So I thing, this addon could be the right for me?!
After my fist tests, I've seen that the weekly group, only tracks the professioin quests and drops?! That's why I've created custom quest tracker for each currently active gold wq. Is this the way to go? And if so, is there an option to group these custom quests so they are displayed like the weekly quest group? So I can see only the total of the completed quests und the individual ones listed in the tooltip?

**Comment by LiangYuxuan** [comment, marked as answer]
You can track custom quest by yourself in config (SI config > Quest Progress > User).
